src/test/routes/user/updateUser.test.ts

```diff
@@ -0,0 +1,162 @@
+import { describe, test, expect, beforeAll, afterAll } from '@jest/globals';
+import { FastifyInstance } from 'fastify';
+import { knexController } from '../../../database/database';
+import { buildServer } from '../../../server';
+import { createTestDatabase, dropTestDatabase } from '../../../lib/testUtils';
+
+describe('update user', () => {
+  const getTestInstance = async (): Promise<FastifyInstance> =>
+    buildServer({
+      routePrefix: 'api',
+    });
+
+  beforeAll(async () => {
+    await createTestDatabase('update_user');
+  });
+
+  afterAll(async () => {
+    await dropTestDatabase();
+    await knexController.destroy();
+  });
+
+  const updatedUser = {
+    email: 'change@email.fi',
+    forename: 'Edited',
+    surname: 'Change',
+    isAdmin: true,
+    isBlender: false,
+  };
+
+  test('it returns obj with updated values', async () => {
```

Comment:
```suggestion
  test('it returns user with updated values', async () => {
```

Vähän liian pitkään katoin tätä riviä ja mietin että miksi täällä mikään palauttaisi Wavefront obj filen 😃